script.myepisodes/default.py

```diff
@@ -124,7 +124,12 @@ def _addShow(self):
             added = 32925
         utils.notif("%s %s" % (self.title, _language(added)))
 
+    # For backward compatibility
     def onPlayBackStarted(self):
+        self.onAVStarted()
```

Comment:
As Leia still sends `onPlayBackStarted` callbacks, this call has to be limited for Krypton only.
Have a look at #977 and #981 how that can be implemented.

Reply:
Ah crap yeah ! Thanks for the tips ! I'll do that then.